dds/InfoRepo/FederatorManagerImpl.h

```diff
@@ -137,8 +137,6 @@ class OpenDDS_Federator_Export ManagerImpl
   //
   // UpdateProcessor<> methods.
   //
-  // "using" directive to fix "Hides the virtual function in virtual base" warning on
-  // SunOS compiler.
   using UpdateProcessor<OwnerUpdate>::processUpdateQos2;
```

Comment:
Should these be removed if they were put in place for the Sun compiler?

Reply:
Looks like Apple clang complains about it as well.